src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
-          ```
+    ```
+    kubectl config set-cluster <var>DESIRED_CLUSTER</var>
+    ```
+
+    To see the available clusters, run `kubectl config get-clusters`
+
+    2. Make sure that `kube-state-metrics` is installed on your machine:
+
+    ```
+    kubectl get deployment --all-namespaces | grep kube-state-metrics
+    ```
+
+    If it's not installed, follow the [instructions in the kube-state-metrics GitHub
+    repo](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment) to install it.
+
+    3. If your cluster uses role-based access, create a service account for `tiller` (Helm's service which runs inside
+    the Kubernetes cluster) using a Kubernetes manifest file:
+
+    4. Paste the following in a manifest file `rbac-config.yaml`:
+
+    ```
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+    name: tiller
+    namespace: kube-system
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+    name: tiller
+    roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-admin
+    subjects:
+    kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+    ```
+    2. Create the service account and role by running:
+
+    ```
+    kubectl create -f rbac-config.yaml
+    ```
     3. Initialize Helm so that `tiller` is installed in the cluster:
 
-       ```
-       helm init --history-max 200
-       ```
+    ```
+    helm init --history-max 200
+    ```
     4. Add the New Relic Helm charts repo:
 
-       ```
-       helm repo add newrelic https://helm-charts.newrelic.com
-       ```
-    5. Make sure everything is configured properly in the chart by running the following command. (This step uses the `--dry-run` and `--debug` switches and therefore the agent is not installed.)
-
-       ```
-       helm install newrelic/newrelic-infrastructure \
-           --dry-run \
-           --debug \
-           --set licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
-           --set cluster=<var>K8S_CLUSTER_NAME</var> \
-           --set config.custom_attributes.cluster=<var>K8S_CLUSTER_NAME</var>
-       ```
+    ```
+    helm repo add newrelic https://helm-charts.newrelic.com
+    ```
+    5. Make sure everything is configured properly in the chart by running the following command. This step uses the
+    `--dry-run` and `--debug` switches and therefore the agent is not installed.
+
+    ```
+    helm install newrelic/newrelic-infrastructure \
+    --dry-run \
+    --debug \
+    --set licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+    --set cluster=<var>K8S_CLUSTER_NAME</var> \
+    --set config.custom_attributes.cluster=<var>K8S_CLUSTER_NAME</var>
+    ```
     6. Install the New Relic Kubernetes integration:
 
-       <Callout variant="important">
-         Note that the test switches have been removed.
-       </Callout>
+    <Callout variant="important">
+      Note that the test switches have been removed.
```

Comment:
?

Reply:
Not sure why it was like this, I guess to clarify why the code snippet is repeated again. I've changed it to a tip to not give false sense of alarm.